cmd/scollector/collectors/database.go

```diff
@@ -0,0 +1,345 @@
+package collectors
+
+import (
+	"database/sql"
+	"fmt"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	_ "bosun.org/_third_party/github.com/go-sql-driver/mysql"
+	"bosun.org/cmd/scollector/conf"
+	"bosun.org/collect"
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+	"bosun.org/slog"
+)
+
+type DatabaseQuery struct {
```

Comment:
Why are is there a `conf.DatabaseQuery` and a `collectors.DatabaseQuery` struct? Just use the conf one everywhere.